Mathlib/MeasureTheory/MeasurableSpace.lean

```diff
@@ -388,6 +389,7 @@ instance Bool.instMeasurableSingletonClass : MeasurableSingletonClass Bool := 
 #align bool.measurable_singleton_class Bool.instMeasurableSingletonClass
 instance Nat.instMeasurableSingletonClass : MeasurableSingletonClass ℕ := ⟨fun _ => trivial⟩
 #align nat.measurable_singleton_class Nat.instMeasurableSingletonClass
+instance Fin.instMeasurableSingletonClass n : MeasurableSingletonClass (Fin n) := ⟨fun _ => trivial⟩
```

Comment:
I've never seen this syntax before: can we go with the clearer
```suggestion
instance Fin.instMeasurableSingletonClass (n : ℕ) : MeasurableSingletonClass (Fin n) := ⟨fun _ => trivial⟩
```
(and below)

Reply:
Sure!